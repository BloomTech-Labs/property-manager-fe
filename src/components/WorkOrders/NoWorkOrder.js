/* eslint-disable react/prop-types */
import React from 'react';
import { Paper } from '@material-ui/core';

function NoWorkOrder() {
  return (
    <>
      <Paper>
        <h4 className="noWorkOrders">
          You don&apos;t have any active Work Orders yet!
        </h4>
      </Paper>
    </>
  );
}

export default NoWorkOrder;
