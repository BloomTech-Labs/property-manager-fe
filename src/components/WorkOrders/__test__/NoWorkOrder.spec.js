import React from 'react';
import * as rtl from '@testing-library/react';
import '@testing-library/jest-dom';
import NoWorkOrder from '../NoWorkOrder';

afterEach(rtl.cleanup);

test('Makes sure the word work order exists', async () => {
  const wrapper = rtl.render(<NoWorkOrder />);
  expect(await wrapper.queryAllByText(/work order/i));
});

test('snapshot', () => {
  const { container } = rtl.render(<NoWorkOrder />);
  expect(container.firstChild).toMatchSnapshot();
});
