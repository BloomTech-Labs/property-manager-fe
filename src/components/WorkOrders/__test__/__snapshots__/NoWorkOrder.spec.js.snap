// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot 1`] = `
<div
  class="MuiPaper-root MuiPaper-elevation1 MuiPaper-rounded"
>
  <h4
    class="noWorkOrders"
  >
    You don't have any active Work Orders yet!
  </h4>
</div>
`;
